analysis/fake_data_stan_analysis.py

```diff
@@ -0,0 +1,391 @@
+#
+# fake_data_stan_analysis.py
+# Author: T. E. Weiss
+# Date modified: May 3, 2020
+#
+# This script generates fake data, then analyzes the data in Stan to infer posteriors.
+# Pathnames configured for running from: scripts/Phase-II-official-analysis/fake-data-study/
+#
+
+"""
+To-do:
+    - In FakeExperimentEnsemble, add:
+        1. Tracking of convergence issues, so that a summary of problems can be saved/printed
+        2. An option to parallelize with slurm instead of multiprocessing
+    - Run morpho processors a) for pre-generation sampling and b) for continuous-variable calibration (once the processors have been tested sufficiently)
+    - Improve some comments (e.g. describing Stan priors)
+"""
+
+
+#import unittest
+from morpho.utilities import morphologging
+logger = morphologging.getLogger(__name__)
+import matplotlib.pyplot as plt
+
+#Importing constants from mermithid
+from mermithid.misc.Constants import *
+from mermithid.processors.TritiumSpectrum.FakeDataGenerator import FakeDataGenerator
+#Importing processors from morpho
+from morpho.processors.sampling import PyStanSamplingProcessor, PriorSamplingProcessor
+from morpho.processors.plots import APosterioriDistribution, Histo2dDivergence
+from morpho.processors.IO import IOROOTProcessor, IORProcessor
+
+#Defining processors
+priorSampler = PriorSamplingProcessor("sample")
+specGen = FakeDataGenerator("specGen")
+writerProcessor = IOROOTProcessor("writer")
+readerProcessor = IOROOTProcessor("reader")
+rReaderProcessor = IORProcessor("reader")
+analysisProcessor = PyStanSamplingProcessor("analyzer")
+aposterioriPlotter = APosterioriDistribution("posterioriDistrib")
+divPlotter = Histo2dDivergence("2dDivergence")
+
+
+def DefineGeneratorInputs(root_file='./results/tritium_analysis.root'):
+    """
+    Samples inputs to a fake data generator from priors, then combines them in a dictionary with fixed inputs to the generator. Saves all the inputs to a root file.
+    
+    Returns:
+        generator_inputs: dictionary of all inputs to a fake data generator
+    """
+    prior_sampler_config = {
```

Comment:
Yes. I don't know how termite works, but I think it would be great if all the hard-coded numbers in this script were put in a configuration file, then read in/inputted to the functions. Would we use a yaml file, like the ones in termite's prod_katydid_config folder?

Are we planning to use termite as opposed to the scripts repository because termite files are more official?